src/components/form-helpers/form-helpers.js

```diff
@@ -556,3 +556,35 @@ const _SubmitButton = ({ children, formik: { isSubmitting }, className }) => {
 };
 
 export const SubmitButton = connect(_SubmitButton);
+
+export const SelectFieldItem = ({
+  className,
+  fieldName,
+  label,
+  type,
+  placeholder,
+  required,
+  values,
+  ...rest
+}) => (
+  <FieldItem className={concatClasses('field-item', className)} fieldName={fieldName}>
+    <label htmlFor={fieldName}>{label}</label>
+    <span className="dropdown-arrow"></span>
+    <Field
+      as="select"
+      name={fieldName}
+      id={fieldName}
+      placeholder={placeholder}
+      validate={createRequiredValidation(required)}
+      {...rest}
+    >
+      {values.map(function (item) {
```

Comment:
Pudieses usar arrows functions, es mas corto
```suggestion
{values.map((item) => (
          <option key={item.key} value={item.key}>
            {item.value}
          </option>
        )
      )}
```